src/components/HomePage/ProductPlacementArea/productDescriptionStyles.js

```diff
@@ -0,0 +1,45 @@
+/* istanbul ignore file */
+import { makeStyles } from "@material-ui/core/styles";
+
+export default makeStyles(() => ({
+    productDescription: ({ isMobile }) => ({
+        display: "flex",
+        flexDirection: isMobile ? "column" : "row",
+        justifyContent: "center",
+        width: isMobile ? "100%" : "80%",
+        margin: "5em auto",
+    }),
+    productDescriptionCol: ({ isMobile }) => ({
+        flex: "1 1 0",
+        padding: "2em",
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "space-between",
+        alignItems: "center",
+        "& > svg": {
+            width: "125px",
+            height: "auto",
+            marginBottom: "2em",
+        },
+        "& > div": {
+            flexGrow: "1",
+            width: isMobile ? "100%" : "75%",
+            textAlign: "center",
+        },
+    }),
+    productDescriptionInfo: {
+        display: "flex",
+        flexDirection: "column",
+        "& > p:not(:first-child)": {
+            margin: "1em 0 0.5em 0",
+        },
+    },
+    productDescriptionAnchor: {
+        width: "100%",
+        height: "100%",
+        "&:hover": {
+            textDecoration: "none",
+        },
```

Comment:
Instead of doing most of this "by hand", which is harder to maintain, you should refactor and use [Grid](https://material-ui.com/components/grid/). ProductDescription would be a grid container, and each column would be a grid item with xs=12 (this makes each column take the full space in mobile (xs)) and md=6 (taking only half space in md size and above).

You might want to experiment with the thresholds to see what fits best (using `sm` instead of `md`, for example, but the base is this one) Then you'll only need to give the main component (product description) some styles to have bigger side margins to keep it similar to what it is now and center it (I don't know if it will center automatically) -> you can just wrap the grid container in another flex div, and add `justify-content: center`, without changing `flex-direction`